ci/build-static-binutils.sh

```diff
@@ -18,10 +18,6 @@ case "$ARCH" in
         export CFLAGS="-m32"
         export CXXFLAGS="-m32"
         ;;
-    *)
```

Comment:
This check should not be removed. If anything, it should be extended.

Cross-compilation must not be removed.